### PR TITLE
Make mincFDR.mincSingleDim use fdr method by default, fixes #189

### LINKED ISSUE #189: mincFDR.mincSingleDim uses qvalue as default method
This should be "fdr" like the rest of them

### PR DESCRIPTION
**Pull request by gdevenyi**

### CODE REVIEW

**Comment by cfhammill** [comment]
This should issue a message that the default has changed

**Comment by gdevenyi** [comment]
Added warning regarding change in method.

**Comment by gdevenyi** [comment]
All tests failing on getting the RMINC test data. Certificate issues?

**Comment by cfhammill** [comment]
Yup, fixed the certificate issues and trigger a re-run

File: R/minc_FDR.R [hunk 1]
```diff
@@ -58,10 +58,11 @@ mincFDR <- function(buffer, ...) {
 }
 #' @describeIn mincFDR mincSingleDim
 #' @export
-mincFDR.mincSingleDim <- function(buffer, df, mask = NULL, method = "qvalue", ...) {
+mincFDR.mincSingleDim <- function(buffer, df, mask = NULL, method = "fdr", ...) {
   if (is.null(df)) {
     stop("Error: need to specify the degrees of freedom")
   }
+  warning('default method has changed, specify method = "qvalue" to retain old implementation')
```
Comment (cfhammill) [inline R/minc_FDR.R]:
```suggestion
  message('default method has changed, specify method = "qvalue" to retain old implementation')
```

Comment (gdevenyi) [reply R/minc_FDR.R]: Okay, but why not warning? Too strong?

Comment (cfhammill) [reply R/minc_FDR.R]: Yeah I think so. I'm of the view most warnings should be errors, CRAN for example will fail you if your test suite has any warnings. Also warnings can be promoted to errors with `options(warn = 2)`, a deprecation message should never terminate your program imo.

Comment (gdevenyi) [reply R/minc_FDR.R]: Cool, makes sense.

**Comment by gdevenyi** [comment]
Hrm, checks still failing...

**Comment by cfhammill** [comment]
Nah it's fine, something loopy with the mac builds on travis at the moment, but the linux build passed.

**Comment by cfhammill** [comment]
Oops, can you close this and PR against develop? I should have caught that earlier, my fault.

**Pull request closed without merging**